src/sql/workbench/parts/editData/browser/editData.component.ts

```diff
@@ -409,7 +409,7 @@ export class EditDataComponent extends GridParentComponent implements OnInit, On
 	 * Replace the line breaks with space.
 	 */
 	private spacefyLinebreaks(inputStr: string): string {
```

Comment:
The name and description aren't really accurate anymore - maybe something like replaceLinebreaks?